config/settings.yml

```diff
@@ -96,6 +96,7 @@ aws:
   access: <%= ENV['AWS_ACCESS_KEY'] %>
   secret: <%= ENV['SECRET_ACCESS_KEY'] %>
   queue: <%= ENV['AWS_QUEUE_NAME'] %>
```

Comment:
perhaps we should explicitly call this something like `request_queue`, `submitted_queue`, relative to the `response_queue`